### DISCUSSION TITLE
Работа Zapret на Nobara

### DISCUSSION THREAD

**Discussion by EfremchickEga** [bol-van/zapret#1005] [Q&A]
Здравствуйте скажите пожалуйста будет ли Zapret работать на Nobara Linux и если да то как его настроить?

**Comment by bol-van** [comment]
Гугл показывает, что он fedora based. На fedora должно работать.
Если на этом все как на федоре, используется systemd или openrc, то должно взлететь
Как настроить описано в docs/quick_start.txt
